app/views/users/new.html.erb

```diff
@@ -0,0 +1,39 @@
+</h1>Create a new user </>
+<% 
+=begin %>
+ <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
- [ ] Here, you should be using the `users_path` helper for action

![Screen Shot 2020-11-18 at 6 32 52 PM](https://user-images.githubusercontent.com/48804258/99565885-78b30f00-29cc-11eb-8bba-147e125c45fe.png)